README.md

```diff
@@ -22,10 +22,15 @@ var MongoStore = require('rate-limit-mongo');
 
 var limiter = new RateLimit({
   store: new MongoStore({
-    // see Configuration
+    uri: "mongodb://127.0.0.1:27017/test_db",
+    user: "mongouser",
+    password: "mongopassword",
```

Comment:
I debated on making this a `process.env.MONGO_PASSWORD` - that's definitely better practice, but if I did, then I'd probably also make `uri` and `user` into env vars, and at that point it's lost some of it's simplicity. Up to you, though.

Reply:
I believe that the way of providing settings for the app is up to the app developer